interfaces/cython/cantera/examples/onedim/diffusion_flame.py

```diff
@@ -22,7 +22,7 @@
 
 # Create the gas object used to evaluate all thermodynamic, kinetic, and
 # transport properties.
-gas = ct.Solution('gri30.xml', 'gri30_mix')
+gas = ct.Solution('gri30.yaml', 'gri30')
```

Comment:
If I have my druthers (a big if 😄 ), the three `gri30` phases will remain in this input file, so this phase will not have a transport model enabled, so the `transport_model` kwarg should be passed here, right?

Reply:
The default transport model for YAML is `Mix` (which is different from the XML version, which didn't have a default tranport model for the 'gri30` phase).

Reply:
Cherry-picked the commit replacing `gri30_mix`, which is now part of PR #737 (which is a better venue for resolution of #736).